lib/file_transfer_component/commands/initiate.rb

```diff
@@ -6,6 +6,8 @@ class Initiate
 
       initializer :name, :uri
 
+      dependency :identifier, Identifier::UUID::Random
```

Comment:
Don't forget to assign an instance of the Udentifier::UUID::Random to this dependency attribute in the build method